javalanglib/src/main/scala/java/lang/Integer.scala

```diff
@@ -54,6 +54,16 @@ object Integer {
   final val MAX_VALUE = 2147483647
   final val SIZE = 32
 
+  /**
+   * The digits for every supported radix.
+   */
+  // Ported from https://android.googlesource.com/platform/libcore/+/master/luni/src/main/java/java/lang/IntegralToString.java
+  private val DIGITS: Array[Char] = Array[Char](
+    '0', '1', '2', '3', '4', '5', '6', '7', '8', '9',
+    'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j',
+    'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't',
+    'u', 'v', 'w', 'x', 'y', 'z')
+
```

Comment:
This won't probably be needed anymore. Otherwise, consider writing this as: `('0' to '9' ++ 'a' to 'z').toArray`